packages/x-date-pickers/src/PickersShortcuts/PickersShortcuts.tsx

```diff
@@ -85,7 +85,12 @@ function PickersShortcuts<TValue>(props: PickersShortcutsProps<TValue>) {
         {
           maxHeight: VIEW_HEIGHT,
           maxWidth: 200,
-          overflow: 'auto',
+          scrollbarGutter: 'stable',
```

Comment:
This feature is under a feature flag on Safari, but I wouldn't say the behavior is bad as is. 👌